test/functional/basic/gesture-e2e-specs.js

```diff
@@ -1,310 +1,362 @@
+            .up();
+
+          await driver.actions([thumb, foreFinger]);
+        }
+        await doPinch();
+      });
+    });
+    describe('special actions', function () {
+      // TODO: Neither ControlCenterView or ControlCenterView is available on recent iOS.
+      it.skip('should open the control center', async function () {
+        let isStatusBarAvailable = false;
+        try {
+          await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeStatusBar'`)
+            .should.eventually.be.rejectedWith(/An element could not be located/);
+        } catch (err) {
+          // if this exists,
+          isStatusBarAvailable = true;
+          await driver.$('~ControlCenterView')
+            .should.eventually.be.rejectedWith(/An element could not be located/);
+        }
+
+        let x, y0, y1;
+        const window = await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeApplication'`);
+        const {width, height} = await window.getSize();
+        try {
+          // Try locating the 'Cellular' element (which can be pulled down)
+          const cellularEl = await driver.$('~Cellular');
+          const location = await cellularEl.getLocation();
+          x = location.x;
+          y0 = location.y;
+        } catch (e) {
+          // Otherwise, pull down the middle of the top of the Simulator
+          const isIphoneX = await (async () => {
+            if (UICATALOG_CAPS.deviceName.toLowerCase().includes('iphone x')) {
+              return true;
+            }
+            const { platformVersion, deviceName } = await driver.sessionCapabilities();
+            const translatedDeviceName = translateDeviceName(platformVersion, deviceName).toLowerCase();
+            return _.includes(translatedDeviceName, 'iphone x');
+          })();
+
+          x = width / 2;
+          y0 = isIphoneX
+            ? 15
+            : height - 5;
+        }
+        y1 = height / 2;
+
+        await driver.touchAction([
+          { action: 'press', x, y: y0 },
+          { action: 'moveTo', x, y: y1 },
+        ]);
+
+        // Control Center ought to be visible now
+        if (isStatusBarAvailable) {
+          await driver.$('~ControlCenterView');
+        } else {
+          await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeStatusBar'`);
+        }
+      });
+    });
+  });
+  describe('tap with tapWithShortPressDuration cap', function () {
+    // needs a special cap, so has to be in its own session
+    before(async function () {
+      const caps = amendCapabilities(UICATALOG_CAPS, {
+        'appium:tapWithShortPressDuration': 0.01,
+        'appium:usePrebuiltWDA': hasDefaultPrebuiltWDA(),
+      });
+      driver = await initSession(caps);
+    });
+    after(async function () {
+      await deleteSession();
+    });
+
+    it.skip('should tap on the element', async function () {
+      let el1 = await driver.$('~Alert Views');
+      await driver.touchAction({
```

Comment:
not sure if touchAction is present in w3c spec

Reply:
webdriver seems to implement touchAction using [touch/perform of JSONWP](https://appium.io/docs/en/commands/interactions/touch/touch-perform/).
I'm not sure if you want to keep tests for them, I just separated [actions](https://github.com/appium/appium-xcuitest-driver/pull/1479/files/15937b4c3ee6e00fe7a2d7fb1e3edba55b2b69df#diff-2328317130b17142ab2be883fa2df7bde2d086f9341a6099b1b06f28b0ad0521R64) and [touchAction](https://github.com/appium/appium-xcuitest-driver/pull/1479/files/15937b4c3ee6e00fe7a2d7fb1e3edba55b2b69df#diff-2328317130b17142ab2be883fa2df7bde2d086f9341a6099b1b06f28b0ad0521R91).
This line will be removed accordance with [this comment](https://github.com/appium/appium-xcuitest-driver/pull/1479#discussion_r1058298739).

Reply:
lets keep it for now, but I would prefer to eventually get rid of all the jwp-specific stuff